jkube-kit/resource/helm/src/main/java/org/eclipse/jkube/kit/resource/helm/HelmParameter.java

```diff
@@ -28,8 +28,16 @@ public HelmParameter(Parameter parameter) {
   public Parameter getParameter() {
     return parameter;
   }
-
+  
   public String getHelmName() {
     return parameter.getName().toLowerCase();
   }
+
+  public String getHelmName(final boolean preserveCase) {
+    if(preserveCase) {
+      return parameter.getName();
+    } else {
+      return parameter.getName().toLowerCase();
+    }
```

Comment:
This just my opinion, but we should try to use original method here:

```java
  public String getHelmName(final boolean preserveCase) {
    if (preserveCase) {
      return parameter.getName();
    }
    return getHelmName();
  }
```

Reply:
Switched to use the original method as suggested.